pkg/api/types.go

```diff
@@ -2033,9 +2035,39 @@ const (
 	// Number of Pods that may be running on this Node: see ResourcePods
 )
 
+// ResourceNameList is a collection of ResourceName values.
+type ResourceNameList []ResourceName
```

Comment:
The sort functions should go in a separate package.  Probably somewhere under a util.  That said, since a resource name is just a string can we just cast to a string slice and use sort strings?